Symfony/CS/Tests/Fixer/Contrib/ModernizeTypesCastingFixerTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Contrib;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author Vladimir Reznichenko <kalessil@gmail.com>
+ */
+class ModernizeTypesCastingFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @dataProvider provideExamples
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideExamples()
+    {
+        $multiLinePatternToFix = <<<'FIX'
+<?php $x =
+intval
+
+(
+    mt_rand
+    (
+        0, 100
+    )
+
+)
+
+;
+FIX;
+        $multiLinePatternFixed = <<<'FIXED'
+<?php $x =
+(int) (
+    mt_rand
+    (
+        0, 100
+    )
+
+)
+
+;
+FIXED;
+
+        return array(
+            array('<?php $x = "intval";'),
+
+            array('<?php $x = ClassA::intval(mt_rand(0, 100));'),
+            array('<?php $x = ScopeA\\intval(mt_rand(0, 100));'),
+            array('<?php $x = namespace\\intval(mt_rand(0, 100));'),
+            array('<?php $x = $object->intval(mt_rand(0, 100));'),
+
+            array('<?php $x = new \\intval(mt_rand(0, 100));'),
+            array('<?php $x = new intval(mt_rand(0, 100));'),
+            array('<?php $x = new ScopeB\\intval(mt_rand(0, 100));'),
+
+            array('<?php intvalSmth(mt_rand(0, 100));'),
+            array('<?php smth_intval(mt_rand(0, 100));'),
+
+            array('<?php "SELECT ... intval(mt_rand(0, 100)) ...";'),
+            array('<?php "SELECT ... intval(mt_rand(0, 100)) ...";'),
+            array('<?php "test" . "intval" . "in concatenation";'),
+
+            array('<?php $x = intval($x, 16);'),
+            array('<?php $x = intval($x, $options["base"]);'),
+            array('<?php $x = intval($x, $options->get("base", 16));'),
+
+            array('<?php $x = (int) $x;', '<?php $x = intval($x);'),
+            array('<?php $x = (float) $x;', '<?php $x = floatval($x);'),
+            array('<?php $x = (float) $x;', '<?php $x = doubleval($x);'),
+            array('<?php $x = (string) $x;', '<?php $x = strval($x);'),
+            array('<?php $x = (bool) $x;', '<?php $x = boolval($x);'),
+            array('<?php $x = (int) (mt_rand(0, 100));', '<?php $x = intval(mt_rand(0, 100));'),
+            array('<?php $x = (int) (mt_rand(0, 100));', '<?php $x = \\intval(mt_rand(0, 100));'),
+            array('<?php $x = (int) (mt_rand(0, 100)).".dist";', '<?php $x = intval(mt_rand(0, 100)).".dist";'),
+            array('<?php $x = (int) (mt_rand(0, 100)).".dist";', '<?php $x = \\intval(mt_rand(0, 100)).".dist";'),
```

Comment:
I meant tests for `floatval`, `doubleval`, `strval`, `boolval`. For now fixer changes all functions but tests cover only one.